cime_config/machines/config_machines.xml

```diff
@@ -2801,7 +2801,7 @@
       <executable>/gpfs/alpine/world-shared/cli115/mpirun.summit</executable>
       <!-- <executable>jsrun</executable> -->
       <arguments>
-        <arg name="num_tasks"> -n {{ total_tasks }} -N {{ tasks_per_node }}</arg>
+        <arg name="num_tasks"> -n {{ total_tasks }} -N {{ tasks_per_node }} -gpu </arg>
```

Comment:
This will use the GPU setup for all IBM cases. I think it would be wise to create an ibm compiler for CPU runs and an ibmgpu compiler for GPU runs like we do with other comiplers.